### DISCUSSION TITLE
Analytics error when someone fork my repositorie.

### DISCUSSION THREAD

**Discussion by Pudge1996** [vercel/community#2599] [Help] [answered]
### Summary

Hey dude, there's something wrong with my website data, both on GA and Vercel Analytics.

I deployed my site using Vercel Deployments, which is linked to my Github repository. I turned on Vercel Analytics, connected my GA code, and everything seemed perfect.

However, I noticed something odd when someone forked my Github repository and published their website online. Now, I can view their website data in my Vercel Analytics and GA.

What should I do? I've already switched my Github repository from public to private, but that didn't solve the issue.

And he say he did not use my GA code in his code.

Here is my site: https://lrd.im
Here is my repo link(private): https://github.com/Pudge1996/NEXT.LRD.IM

Here is the forker site: https://qquan.cc/

Here is the screenshot from my GA account: (sorry for use Chinese word but annotation was marked.)
![image](https://github.com/vercel/community/assets/22073891/7fcbe1a6-e0dd-4739-bbaa-e0156f1fa3c6)

### Steps to Reproduce

1. Use Vercel deploy a website,
2. Public your Github repo,
3. Someone forked your repo,
4. And his website data will sync to your Vercel Analytics.

**Comment by amyegan** [comment]
Hi @Pudge1996! It looks like the screenshot you shared is from Google Analytics rather than Vercel Analytics. If you included your Google Analytics tag id in the rep, than any forks of the repo would also send data to your Google dashboard.

**Reply by Pudge1996** [reply]
Thank you for your kind response, Amyegan.

Actually, it also made my Vercel Analytics data incorrect.

However, we cannot find the site name in Vercel Analytics, so from this aspect, I can't provide any more details.

But I can assure you that normally, it's impossible for my site to get this many Windows users, as you can see **in my third screenshot.** 

### Current:

And I tried talking to the forker guy, he told me that he has already deleted the file `gtag.js`, and add his personal GA CODE to `_app.js`. 

I am not sure it works or not.

But it seems like a terrible defect, doesn't it? Someone forked my repo, and the data getting dirty...🤔

### Screenshots

Here are screenshots illustrating the incorrect data:

![企业微信截图_271b2ee0-45dd-423e-a433-d9fb9896b5b9](https://github.com/vercel/community/assets/22073891/52a5f4f0-565c-440d-b1ac-5b1d437f89df)

![企业微信截图_e5d903bd-0b5d-4c5e-873b-7c016dec66e3](https://github.com/vercel/community/assets/22073891/1d7abd61-5e0b-4f0f-9dbe-3154dc81774d)

<img width="1290" alt="image" src="https://github.com/vercel/community/assets/22073891/9fb7bf7d-945a-4d85-b62b-f33772c9526c">

**Reply by Pudge1996** [reply, marked as answer]
Update: this guy told me that: "My website is deployed in a Docker container on NAS, and I saved your online page directly to my local machine. Then I made some minor changes to the relevant characters before deploying it on NAS for testing."
